code/game/gamemodes/nuclear/pinpointer.dm

```diff
@@ -225,35 +290,40 @@
 		worklocation()
 		playsound(loc, 'sound/machines/twobeep.ogg', 50, 1)	//Plays a beep
 		visible_message("Shuttle Locator mode actived.")			//Lets the mob holding it know that the mode has changed
+		active = 1
 		return		//Get outta here
 	scandisk()
-	if(!the_disk)
-		icon_state = icon_null
+	point_at(the_disk)
+
+/obj/item/pinpointer/nukeop/workbomb()
+	if(active != 2)
```

Comment:
I'd not check the active/mode state in these. You've already done that prior

Reply:
removed